fizzbuzz_extended/spec/fizzbuzz_extended_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'fizzbuzz_extended'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.